zulip_bots/zulip_bots/lib.py

```diff
@@ -106,6 +109,17 @@ def get_config_info(self, bot_name, section=None, optional=False):
             raise
         return dict(config.items(section))
 
+    @contextmanager
+    def open(self, filepath):
+        # type: (str) -> None
+        filepath = os.path.normpath(filepath)
+        abs_filepath = os.path.join(self._bot_dir, filepath)
+        if abs_filepath.startswith(self._bot_dir):
+            yield open(abs_filepath)
```

Comment:
You don't really need to make this a contextmanager, since you're not really wrapping any behavior within this function.  Instead, you can simply return the other context manager, so change `yield` to `return`.